.github/workflows/docker_publish.yml

```diff
@@ -5,86 +5,16 @@ on:
   workflow_dispatch:
   push:
     paths:
-      - 'CI/?ocker**'
+      - 'CI/Dockerfile'
       - '.github/workflows/docker_publish.yml'
 
-
 jobs:
-  build-base-img:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        ubuntu_versions : [
-          18.04,
-          20.04,
-          ]
-    name: Ubuntu ${{ matrix.ubuntu_versions }} basics deps
-
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-      
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
-
-      - name: Log in to the Container registry
-        uses: docker/login-action@v1
-        with:
-          registry: ghcr.io
-          username: ${{ github.repository_owner }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Build and Export Dockerfile_0_base
-        uses: docker/build-push-action@v2
-        with:
-          file: CI/Dockerfile
-          target: base
-          push: true
-          context: .
-          build-args: |
-            UBUNTU_VERSION=${{ matrix.ubuntu_versions }}
-          tags: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}:ci_testing
-
-  build-housekeeping-img:
-    needs: build-base-img
+  build-test-dagmc-img:
```

Comment:
rename based on comments below

Reply:
this has now been renamed to ```build-dependency-img``` so I think this is resolved